### ISSUE TITLE
Getting a list of contents does not recognize pseudo directories

### ISSUE THREAD

**Issue by macdabby** [rackspace/php-opencloud#627]
It appears the code is checking for a subdir property that does not exist. Instead there is a content type of 'application/directory'

**Comment by jamiehannaford** [comment]
@macdabby Are you using Rackspace or OpenStack?

**Comment by macdabby** [comment]
Yes this was on Rackspace which runs OpenStack from what I understand.
